selfdrive/test/process_replay/regen.py

```diff
@@ -217,6 +253,14 @@ def regen_segment(lr, frs=None, outdir=FAKEDATA):
     'camerad': [
       *cam_procs,
     ],
+    'controlsd': [
```

Comment:
controlsd is supposed to run live. the only things that should be replayed are hw: CAN, cameras, and sensors